sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationEventProcessor/EventResult/PlayEventResult.cs

```diff
@@ -14,14 +14,14 @@ public class PlayEventResult
         /// <summary>
         /// <see cref="PlayCompleted"/> event will be returned once the play is completed successfully.
         /// </summary>
-        public PlayCompleted SuccessResult { get; }
+        public CallAutomationEventBase SuccessResult { get; }
```

Comment:
Why are we changing this to CallAutomationEventBase? These should be the specific events instead of the base event.

Reply:
Is that so? is that a good practice? I was thinking more on the simplification of code instead of its expansion.

Reply:
Yes, we should have seperate event for newely added event.
Also, this would be breaking change - so we would have to do addition